auslib/test/admin/views/test_rules.py

```diff
@@ -590,9 +590,9 @@ def testGetNoRevisions(self):
         got = json.loads(ret.data)
         self.assertEquals(got["count"], 0)
 
-    def testGetRefusesAlias(self):
-        ret = self._get("/rules/frodo/revisions")
-        self.assertEquals(ret.status_code, 404)
+    # def testGetRefusesAlias(self):
```

Comment:
@mozbhearsum,
Is this test currently useful?
The rule query is able to figure out if parameter is an integer or an alias.